openapi/customervoice/customervoice.yaml

```diff
@@ -231,8 +500,121 @@ components:
                       type: string
                       default: template
                       readOnly: true
+    templates:
+      title: Templates
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Templates that can be used for emails and other messaging services.
+      type: object
+      x-examples:
+        Example Template:
+          type: templates
+          id: 'AG-12345/type/email/template-id/TEM-12345'
```

Comment:
Looks like they start with `/account-group-id/AG-12345/...`. Not sure whether you intended to omit that here and prepend it in code or not.
https://vstore-prod.vendasta-internal.com/namespace/templates/table/Template/entity?f=customer-voice&f=

Reply:
It would be hidden from the customer. This format is our internal identifier as "account group" isn't a term that we expose publicly.